lib/CallBackery/GuiPlugin/Users.pm

```diff
@@ -102,6 +103,7 @@ has actionCfg => sub {
             }
         }) : (),
         {
+            key => 'editUser',
```

Comment:
do we NEED keys on action buttons ?

Reply:
Wenn wir sie klicken wollen vom Test-Skript aus schon, denke ich.